regression-tests/generatedata/timeseriesstats.R

```diff
@@ -0,0 +1,31 @@
+goldfile="../datasets/timeseriesstats.txt"
```

Comment:
TeamCity is giving a warning about this file saying unknown file type.  Does this actually run at all with regressions, or is it just for you to know how the tests were run in R to generate that timeseriesstats.txt file?  Can this just be a .txt file?  Either that, or ask Rene about that check to see if he can make it recognize *.R files.